### ISSUE TITLE
Linear classifier from scratch

### ISSUE THREAD

**Issue by bulygin1985** [bulygin1985/ML_CV_study#2]
Task Q3 from cs231n course (https://cs231n.github.io/assignments2024/assignment1/)
Task formulation and "skeleton" of the solution are located here:
https://github.com/bulygin1985/ML_CV_study/blob/main/numpy/assignment1/softmax.ipynb

To Do:
softmax_your_surname.ipynb with your Linear Classifier implementation.

**Comment by bulygin1985** [comment]
How can I obtain gradient of softmax loss function? Is "cross entropy" is unique choice as loss function?
